### PR TITLE
KAT-1974 remove red shadow of required input field for FF

### PR DESCRIPTION
**Pull request by lunxiao**
## PR Checklist

Check these before submitting your pull request:

- [x ] Visual and behavioral components are separated
- [ x] Exported components are documented with examples
- [x ] Props have JSDoc comments
- [x ] All relevant visual sub-components can be overridden via props

## Breaking Changes

* NONE

## Changes to Existing Components

* Changed shadow for required Input component to fix red shadow in FF

### CODE REVIEW

File: src/components/Input/styles/InputStyles.js [hunk 1]
```diff
@@ -24,6 +24,10 @@ const InputStyles = styled.input`
     border-color: ${get('colors.border.medium')};
   }
 
+  &:required {
+    box-shadow: none;
```
Comment (a-type) [inline src/components/Input/styles/InputStyles.js]: This removes the focus effect on Firefox as well by overriding the rule from line 22.

Comment (a-type) [reply src/components/Input/styles/InputStyles.js]: Probably needs to be moved above that rule.

**Comment by a-type** [comment]
I'm concerned that this fix is not going to be very simple, considering it works on Chrome as intended already.

The purpose is not to remove the red effect entirely, only to keep it from appearing until the user has 'visited' the input and left it blank.

**Comment by a-type** [comment]
Just noticed the default invalid state is actually broken in master right now. I have a fix ready: https://github.com/Bandwidth/shared-components/pull/61

File: src/components/Input/styles/InputStyles.js (hunk 1 above)
Comment (lunxiao) [reply src/components/Input/styles/InputStyles.js]: Yeah, I changed the order. Other than that the fix does not change any existing behavior except for removing the red shadow on FF when required fields are initially displayed.

**Pull request merged**